src/files-breadcrumbs.tsx

```diff
@@ -37,18 +37,20 @@ import { basename } from "./common";
 
 const _ = cockpit.gettext;
 
-function BookmarkButton({ path }: { path: string[] }) {
+function BookmarkButton({ path }: { path: string }) {
     const [isOpen, setIsOpen] = React.useState(false);
     const [user, setUser] = React.useState<cockpit.UserInfo | null>(null);
     const [bookmarks, setBookmarks] = React.useState<string[]>([]);
     const [bookmarkHandle, setBookmarkHandle] = React.useState<cockpit.FileHandle<string> | null>(null);
 
     const { addAlert, cwdInfo } = useFilesContext();
 
-    const currentPath = path.join("/") || "/";
     const defaultBookmarks = [];
-    if (user?.home)
-        defaultBookmarks.push({ name: _("Home"), loc: user?.home }); // TODO: add trash
+    if (user?.home) {
+        // Ensure the home dir has a trailing / like the path
+        const home_dir = user.home.endsWith('/') ? user.home : `${user.home}/`;
```

Comment:
This added line is not executed by any test.

Reply:
Yup, never seen `user.home` be /home/admin/ it always is `/home/admin` so that's logical.